Update Readme for reusing python env (#318)
openvinotoolkit/openvino.genai: llm_bench/python/README.md

## Usage 

> ### 1. Start a Python virtual environment Or Reuse an existing python environment

Reviewer: ```suggestion
### 1. Start a Python virtual environment
```

> ### 1. Start a Python virtual environment Or Reuse an existing python environment
> (Recommended) To use a new environment, the steps are as follows:

Reviewer: ```suggestion
### 1. Start a Python virtual environment
```

``` bash
python3 -m venv python-env
source python-env/bin/activate
pip install --upgrade pip
pip install -r requirements.txt
```
> Note:

> > If you want to reuse an existing python exvironment, recommend following command to use all the dependencies with latest versions:  

Reviewer: ```suggestion
> If you are using an existing python environment, recommend following command to use all the dependencies with latest versions:  
```

```
> Note: For llama models, ensure to have transformers<4.38.

If you want to reuse an existing python exvironment, recommend following
command to use all the dependencies with latest versions:
```bash
pip install -U --upgrade-strategy eager -r requirements.txt
```

> 

Reviewer: ```suggestion
> Note:
> If you are using existing python environment, recommend following command to use all the dependencies with latest versions:
> pip install -U --upgrade-strategy eager -r requirements.txt
